spec/index.html

```diff
@@ -515,20 +515,99 @@ <h2>Basic Concepts</h2>
     </p>
 
     <p class="ednote">FIXME.</p>
+
+    <section class="informative" id="convenience-context">
+      <h3>Convenience Context</h3>
```

Comment:
:-1:

Reply:
@ioggstream — It's unclear what you're saying you don't like.  Please elaborate.

Reply:
Since using a context to provide keyword alternates seemed to be a group consensus (I could be wrong, of course), this seems to be the natural consequence of that decision.

In #11, @anatoly-scherbakov laid out a plan for broad replacement of `@-keywords` with `$-keywords`, which would require deeper changes to document loading or core algorithms, and is still fraught with unexpected consequences. I outlined this solution in https://github.com/json-ld/yaml-ld/issues/11#issuecomment-1141352994.

@juusoautiosalo suggested in https://github.com/json-ld/yaml-ld/issues/11#issuecomment-1177200927 to chose a character other than `$`, and if there is consensus over some specific character, we can of course update this PR to use that, instead.

Of course, the ongoing need to quote `"@context"` is not solved by this solution, but seems to be a reasonable tradeoff for using an already standard mechanism for aliasing keywords.

Reply:
On a previous call we'd discussed a "convenience context" but my understanding was that we would simply be providing a context which removed the `@` symbol...not "blessing" a new symbol with magic meaning... Also, given that JSON Schema uses the `$` prefix heavily, attempting to us that runs us into a namespace collision with them (of sorts) and won't make us any friends.

I could support providing a simple context file that aliases things without the `@` prefix (i.e. `"base": "@base`), but developers will have to be aware (as ever) of what that means for the rest of their document/contents.